sharing/android/example/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
A lot of files are missing newlines at the end of file, can you go through and make sure they are all consistent?

Reply:
still seeing the missing newlines?